### DISCUSSION TITLE
Camera Mode Presets for Video - GoPro Hero 11

### DISCUSSION THREAD

**Discussion by compoundvpn** [gopro/labs#819] [General]
GoPro Hero 11, v2.30.70 March 2024 Firmware.

FOR CAMERA VIDEO MODE PRESETS:

Using the following control settings: mV0r5p30fW0e1hS0dR0aWtcNd1b1wAi8M1S0x.5sLa

Manually create the maximum number of custom video mode presets (there are 7). 

Modify each preset with all the same values (for simplicity purposes for this discussion) as shown below:

mV0 modifies 'Standard' preset                   -  mV0r5p30fW0e1hS0dR0aWtcNd1b1wAi8M1S0x.5sLa
mV3 modifies 'Full Frame' preset                 - mV3r5p30fW0e1hS0dR0aWtcNd1b1wAi8M1S0x.5sLa
mv5 modifies the current displayed preset - mV5r5p30fW0e1hS0dR0aWtcNd1b1wAi8M1S0x.5sLa
mV6 modifies the first created Custom preset (from a chosen name or automatically named 'Custom')
mV9 modifies the 4th out of 7 manually created Custom presets.

How can the last 3 custom presets be modified?  

Any way to programatically CREATE the custom presets?

--

FOR CAMERA PHOTO MODE PRESETS:

How can the same be done for the 7 custom presets in Photo Mode? There are no mP0, mP1.. etc commands.

--

Documenting for future addition within the Labs documentation:

CAMERA MODE VIDEO PRESET VALUES (mV)
 
0 - Standard 
1 - Activity
2 - Cinematic
3 - Full Frame
4 - Ultra Slo-Mo
5 - Update current displayed preset

The following presets must be created manually on the camera itself before accessing via mV? (Where '?' is digits 6..9 and ? ? ?)
The first created manually created preset is accessed with mV6, second manually created preset is accessed with mV7, etc
6 - 1st chosen preset (Custom or named)
7 - 2nd chosen preset (Custom 1 or named)
8 - 3rd chosen preset (Custom 2 or named)
9 - 4th chosen preset (Custom 3 or named)
? - 5th chosen preset (Custom 4 or named)
? - 6th chosen preset (Custom 5 or named)
? - 7th chosen preset (Custom 6 or named)

**Comment by dnewman-gpsw** [comment]
The numbers go up.  mV10, mV11 etc.
